source/_docs/ecosystem/nginx.markdown

```diff
@@ -111,6 +111,7 @@ http {
 
         # These shouldn't need to be changed
         listen 443 default_server;
+        listen [::]:443 default_server ipv6only=on;
```

Comment:
I am on my Raspberry Pi 3 running a Debian Jessie derivate (OSMC) and that does not work.

> invalid parameter "http2" in /etc/nginx/sites-enabled/default:44

Also I tested it without `http2` and it does not bind to IPv4. 

I verified `net.ipv6.bindv6only = 0`